README.md

```diff
@@ -39,45 +42,46 @@ To run the integration tests or Python scripts (not required to use the code), i
 Python (3.7 or higher), and install python3 packages using `./scripts/pysync` (requires poetry) in the project directory.
 
 2. Build neon and patched postgres
+
 ```sh
 git clone --recursive https://github.com/neondatabase/neon.git
 cd neon
 make -j5
 ```
 
 3. Start pageserver and postgres on top of it (should be called from repo root):
+
 ```sh
 # Create repository in .zenith with proper paths to binaries and data
 # Later that would be responsibility of a package install script
 > ./target/debug/neon_local init
-initializing tenantid c03ba6b7ad4c5e9cf556f059ade44229
-created initial timeline 5b014a9e41b4b63ce1a1febc04503636 timeline.lsn 0/169C3C8
-created main branch
+initializing tenantid 9ef87a5bf0d92544f6fafeeb3239695c
```

Comment:
Currently, in the latest `main`, running `./target/debug/neon_local init` doesn't print anything. The above output was created by adding a `println!("{}", String::from_utf8_lossy(&init_output.stdout));` statement in https://github.com/neondatabase/neon/blob/main/control_plane/src/storage.rs#L169

Reply:
We'll need to determine if the current behaviour is intended before pushing the change. If it's intended, we should remove the output of `init` command in `REAMDE`

Reply:
@SomeoneToIgnore was that change intentional? And if it wasn't, do we want to change it back, or leave it as it is?

Reply:
I was trying to actualize it up to whatever was the cli showing, feel free to change this however you feel is better.

Reply:
> Great application of Chesterton's Fence :)

TIL, Chesterton's Fence :O. Cool stuff.

> Seems like this is the commit that changed the behavior, since it changed .status() to .output(), capturing the output but not printing it.

I looked into that before. Totally missed the fact the `output` captures `stdout` by default but `status()` doesn't 😅 

IMO, it's good to see the child `pageserver init` command's output so I add it back [here](https://github.com/neondatabase/neon/pull/1649/commits/bc72ca7d45ddebf6eb12d201890341e0f17016d9)